src/components/Invoices/InvoiceCards/index.js

```diff
@@ -0,0 +1,54 @@
+const InvoiceCards = ({invoice}) => {
+    const { due, id, invoice_id, name, invoice_total, status, status_name } = invoice;
+
+    const changeStatusColour = (status) => {
+        const statusMap = {
+            "1": "ps-4 btn btn-outline-success m-0 disabled opacity-100",
+            "2": "ps-4 btn btn-outline-warning m-0 disabled opacity-100",
+            "3": "ps-4 btn btn-outline-danger m-0 disabled opacity-100"
+        };
+        return (
+            <ul className={statusMap[status]}>
+                <li><strong>{status_name}</strong></li>
+            </ul>
+        )
+    }
+
+    const options = {
+        style: 'currency',
+        currency: 'GBP',
+    }
+    const invoicesTotalFloat = parseFloat(invoice_total);
+    const invoicesTotalFormatted = invoicesTotalFloat.toLocaleString("en-GB", options);
+
+    const dateFormatter = (dateString) => {
+        const date = new Date(dateString);
+        const options = {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        };
+        return (new Intl.DateTimeFormat('en-GB', options).format(date));
+    }
+
+    return (
+        <>
+            <div className="bg-white d-flex flex-wrap mb-2 justify-content-between align-items-center p-3 bd-highlight border rounded" key={id}>
+                    <div className="pe-4">#<strong>{invoice_id}</strong></div>
```

Comment:
You can outdent this and the next four to line up with line 42.

Reply:
done